RingPublishingTracking/src/main/java/com/ringpublishing/tracking/data/audio/AudioState.kt

```diff
@@ -0,0 +1,15 @@
+package com.ringpublishing.tracking.data.audio
+
+/**
+ * Audio state data holder
+ * @param [currentTime] - Current audio material timestamp shown in audio player
+ * @param [currentBitrate] - Current audio material bitrate (in kbps)
+ * @param [visibilityState] - Current audio player visibility @see [AudioPlayerVisibilityState]
+ * @param [audioOutput] - Audio output type @see [AudioOutput]
+ */
+data class AudioState(
+    val currentTime: Int,
+    val currentBitrate: String,
```

Comment:
tu nie powinien być int/long?

Reply:
Wcześniej był string, więc tutaj też tak dałem - do dogadania

Reply:
ja bym dał Int własnie z założeniem, że to jest jako kbps